src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -765,5 +776,40 @@ public static void TestingGetStringInvalidUTF8(byte[] dataUtf8)
                 }
             }
         }
+
+        [Theory]
+        [MemberData(nameof(GetCommentTestData))]
+        public static void TestingGetComment(string[] inputData)
```

Comment:
Can you add some more tests similar to the ones that cover `GetString`?
Like:
1) Include escaped characters (verify we are not unescaping)
https://github.com/dotnet/corefx/blob/16024ef86792da180d3b847de24a33d6719e9429/src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs#L646

2) Invalid UTF-8/UTF-16
https://github.com/dotnet/corefx/blob/16024ef86792da180d3b847de24a33d6719e9429/src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs#L724
https://github.com/dotnet/corefx/blob/16024ef86792da180d3b847de24a33d6719e9429/src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs#L746

Reply:
When not unescaping, the invalid UTF-8 / UTF-16 tests do not work because those sequences remain escaped like any other character sequence, but do not resolve invalid UTF-8 / UTF-16 characters.

Reply:
Added test case for point 1.

Reply:
> When not unescaping, the invalid UTF-8 / UTF-16 tests do not work because those sequences remain escaped

Sure, that's fine. I would still add the test strings to validate that behavior (i.e. test the behavior of comments containing such characters).

Reply:
Right. Actually in that case this falls back to point 1 and I have already added two strings - one which contains `\t`, a valid escape sequence and another which contains one invalid UTF16 sequence `\uDD1E`. Please refer [here](https://github.com/dotnet/corefx/pull/35705/commits/ab31be625ff82070fd5a96db7bd86cf1f82c9c32#diff-b12b49879ec5ef146f3373b69e66216bR1949) in the commit that I submitted.

Reply:
Can you add a test case with invalid UTF-8 bytes as well?